src/api-console-styles.html

```diff
@@ -168,6 +167,30 @@
   --raml-request-parameters-editor-subheader: {
     @apply(--arc-font-subhead);
   }
+    /* Anypoint styles */
+    /*Tabs*/
+  --anypoint-paper-tabs-content: {
+    height: calc(100% - 3px);
+    border-bottom: 3px solid rgba(39,47,51,.1);
+    font-family: Open Sans,sans-serif;
```

Comment:
Can we use `--anypoint-font-*` mixins instead of font declaration?
See https://github.com/advanced-rest-client/anypoint-styles/blob/master/typography.html for list of mixins.